src/bb-components-build.ts

```diff
@@ -37,7 +37,7 @@ const readComponents: () => Promise<
   const exists: boolean = await pathExists(srcDir);
 
   if (!exists) {
-    throw new Error(chalk.red('\nComponents folder not found\n'));
+    throw new Error('\nComponents folder not found\n');
```

Comment:
Can you move the formatting to inside the catch.

Reply:
Done!